src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseStatistics.cpp

```diff
@@ -340,6 +360,8 @@ void CompositionalMultiphaseStatistics::computeRegionStatistics( MeshLevel & mes
                                          << ": Phase dynamic pore volumes: " << regionStatistics.phasePoreVolume << " rm^3" );
     GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
                                          << ": Phase mass: " << regionStatistics.phaseMass << " " << massUnit );
+    GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
+                                         << ": Trapped phase mass: " << regionStatistics.trappedPhaseMass << " " << massUnit );
```

Comment:
```suggestion
    GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
                                         << ": Phase mass (trapped and non-trapped): " << regionStatistics.phaseMass << " " << massUnit );
    GEOSX_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
                                         << ": Trapped phase mass: " << regionStatistics.trappedPhaseMass << " " << massUnit );
```
to avoid confusion, since `phase mass` includes some `trapped phase mass`.